dffml/util/data.py

```diff
@@ -215,6 +215,8 @@ def export_value(obj, key, value):
         obj[key] = value.export()
     elif hasattr(value, "_asdict"):
         obj[key] = value._asdict()
+    elif getattr(type(value), "__module__", None) == "numpy":
```

Comment:
We should also check that the `flatten` method exists and can be called to be safe

```suggestion
    elif getattr(type(value), "__module__", None) == "numpy" and inspect.ismethod(getattr(value, "flatten", None)):
```

Reply:
Alright!